proto/blob/tx.proto

```diff
@@ -26,16 +26,16 @@ message ShareCommitAndSignature {
 // MsgPayForBlob pays for the inclusion of a blob in the block.
 message MsgPayForBlob {
```

Comment:
Definitely!! started to change this in this PR, but since this PR is already large, decided to spin it off as a follow up. https://github.com/celestiaorg/celestia-app/issues/1221